lib/samson/clair.rb

```diff
@@ -12,6 +12,7 @@ class << self
       def append_job_with_scan(job, docker_tag)
         return unless clair = ENV['HYPERCLAIR_PATH']
 
+        ActiveRecord::Base.clear_active_connections!
```

Comment:
could this go inside the thread? that way if the bug wasn't triggered it will not actually release anything?